android/app/src/main/java/app/organicmaps/location/LocationHelper.java

```diff
@@ -311,20 +312,36 @@ public void removeListener(@NonNull LocationListener listener)
     mListeners.remove(listener);
   }
 
+// OM now always updates when a new location is available. This is left in-case
+// we want to go back to using different mode intervals.
```

Comment:
Its better to remove the commented code (maybe just leave a short comment about the previous behavior). In case we'd want to come back to it we can revert always.

Reply:
fair, done 👍️